### PR TITLE
Tup#89 Enabling ethereum for old spaces

### PR DESCRIPTION
**Pull request by gdalyy**

### CODE REVIEW

File: jobs/MigrateSpace.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        ]);
+
+        foreach ($transactions as $transaction) {
+            $transactionEvent = new Event(['sender' => $transaction]);
+            if ($transaction->transaction_type == Transaction::TRANSACTION_TYPE_ISSUE) {
+                // mint coins for each issue transaction of the space
+                Coin::mintCoin($transactionEvent);
+            } else {
+                //transfer coins for each coin holder
+                Coin::transferCoin($transactionEvent);
```
Comment (bumi) [inline jobs/MigrateSpace.php]: we should not replicate all transfers but only set the balance. 
doing all transfers might lead to lots of transactions and thus costs. Also it is not really needed for old transactions, is it?

**Review by bumi — CHANGES_REQUESTED** [review]
we should only set the current balances for each user and not replicate all historic transaction.... what are your thoughts @haythem96

File: jobs/MigrateSpace.php (hunk 1 above)
Comment (haythemsellami) [reply jobs/MigrateSpace.php]: Yeah I think so, it will be better to mint the balances for the users...less transactions

Comment (gdalyy) [reply jobs/MigrateSpace.php]: Okay , i'll implement the call for POST /migrate/space to use only `mint` and not replicate all transfers

**Review by haythemsellami — APPROVED** [review]

**Review by bumi — APPROVED** [review]
looks good to me, from what I can tell.

**Pull request merged**